src/workflow.tsx

```diff
@@ -1,12 +1,87 @@
 import * as React from 'react';
-import greet from './greet/greet';
+
+export enum Labels {
+  next = 'next',
+  previous = 'previous',
+}
+
+export interface WorkflowItem {
```

Comment:
What about putting all interfaces in a separated file like workflowTypes.ts? In my opinion it would help to keep the component file more focused on rendering it and its logic, while a workflowTypes.ts would export all these interfaces and types that could be imported by workflow.tsx or any other component that need them.

Reply:
I can see how a single file could get crowded with definitions and actual code, I like this idea 👍